staging/cse/windows/provisioningscripts/windowsnodereset.ps1

```diff
@@ -15,7 +15,8 @@ $global:NetworkMode = "L2Bridge"
 $global:NetworkPlugin = $Global:ClusterConfiguration.Cni.Name
 $global:ContainerRuntime = $Global:ClusterConfiguration.Cri.Name
 $UseContainerD = ($global:ContainerRuntime -eq "containerd")
-$IsDualStackEnabled = $Global:ClusterConfiguration.Kubernetes.Kubeproxy.FeatureGates -contains "IPv6DualStack=true"
+$IsDualStackEnabled = ($Global:ClusterConfiguration.Kubernetes.Kubeproxy.FeatureGates -contains "IPv6DualStack=true") `
+                        -Or ($Global:ClusterConfiguration.Kubernetes.Kubelet.NodeLabels -like "*kubernetes.azure.com/podv6network-type=overlay*")
```

Comment:
open to something else besides checking if this label exists. the FeatureGate won't be set so we'll need something else and this was the best option when I looked at the ClusterConfig json object